admin_panel/admin_panel/admin.py

```diff
@@ -1,8 +1,10 @@
 from django.contrib import admin
 
+from ballsdex.settings import settings
+
 
 class BallsdexAdminSite(admin.AdminSite):
-    site_header = "Ballsdex administration"  # TODO: use configured bot name
-    site_title = "Ballsdex admin panel"
+    site_header = f"{settings.bot_name.title()} administration"
+    site_title = f"{settings.bot_name.title()} admin panel"
```

Comment:
I don't think we need to use title here as that can be done directly via the config setting.

```suggestion
    site_header = f"{settings.bot_name} administration"
    site_title = f"{settings.bot_name} admin panel"
```